lib/pure/unidecode/unidecode.nim

```diff
@@ -66,8 +66,3 @@ proc unidecode*(s: string): string =
     var c = int(r)
     if c <=% 127: add(result, chr(c))
     elif c <% translationTable.len: add(result, translationTable[c-128])
-
-when isMainModule:
-  #loadUnidecodeTable("lib/pure/unidecode/unidecode.dat")
-  doAssert unidecode("Äußerst") == "Ausserst"
-  doAssert unidecode("北京") == "Bei Jing "
```

Comment:
why were those removed?

Reply:
tests already exist
https://github.com/nim-lang/Nim/blob/devel/tests/stdlib/tunidecode.nim

Reply:
i saw it but the ones you removed are missing; yu could simply add back these:
```nim
doAssert unidecode("Äußerst") == "Ausserst"
doAssert unidecode("北京") == "Bei Jing "
```
(and use doAssert instead of echo)

Reply:
https://github.com/nim-lang/Nim/pull/16157